ImmaculateHangover/crunchy-tunes
# Resolves #58 - feat/ Implement live search feature and integrate API requests function

Checklist

Mark an X inside of

[x] Did I cut my work branch off of master (don't cut new branches from existing feature branches)?
[x] Did I follow the correct naming convention for my branch?
[x] Is my branch focused on a single main change?
[x] Do all of my changes directly relate to this change?
[x] Did I rebase the upstream master branch after I finished all my work?
[x] Did I write a clear pull request message detailing what changes I made?
[x] Are all tests passing
[x] Did I write new tests for new functionality?

What are the important parts of the code?

The important parts of this code can be found within the app, cardsContainer, nav, search, and songPlayer modules. We imported the queryAll function to app the app module, and passed it down to the search component. Now, whenever a search is performed, the function runs and updates the cardsContainer view with the relevant search results.

How should this be tested by the reviewer?

Verify that all tests are passing. Also, load the web application, and test the live search functionality by performing a demo live search.

Is any other information necessary to understand the files?

Negative.
